app.js

```diff
@@ -57,6 +66,44 @@ var server = app.listen(port, function () {
     console.log('Server running at http://127.0.0.1:' + port + '/');
 });
 
+//initial meeting endpoints
+
+//create a meeting
+app.post("/meeting", function (req, res) {
+    var start_datetime = req.query.start_datetime;
+    var end_datetime = req.query.end_datetime;
+    var room_number = req.query.room_number;
+
+    console.log(req);
+
+    var sql = "INSERT INTO ebdb.Meeting (start_datetime, end_datetime, room_number) VALUES (?, ?, ?);"
+    var inserts = [start_datetime, end_datetime, room_number];
+    mysql.format(sql, inserts);
+
+    console.log(mysql.format(sql, inserts));
+
+    pool.getConnection(function(err,connection) {
```

Comment:
We should be able to just use ``pool.query( ) `` here.

Reply:
Fixed